### ISSUE TITLE
Batch dates incorrect

### ISSUE THREAD

**Issue by pjw6193** [revaturelabs/caliber#792]
### Issue Description
When importing a batch, many times Caliber will save the new batch with the start date and end date 3-4 days ahead. When selecting the date from the datepicker fields, the start date and end date will always display as a day ahead.

### Expected behavior
Start and end dates for a batch must be accurate.

### Actual behavior (bugs)
One day ahead with manual entry, 3-4 days ahead for importing from Salesforce

### Steps for replicating (bugs)
Create or import a batch

**Comment by erchiggins** [comment]
@pjw6193 this issue still does not work. the start/end date is still one day ahead

**Comment by erchiggins** [comment]
Details about current state: 

"On the angular side, the date was taken and sent in string format (2017-12-05) to the server side. We converted the string using Jackson to a java object. The start and end date are stored as a java.util.date, which has a timezone associated with it. During the conversion to the batch object, the start and end date that was received is assumed to be in GMT time and was converted to EST (GMT-5) timezone. 
Example: startDate=Sun Nov 20 00:00:00 GTM 2017 == startDate=Sun Nov 19 19:00:00 EST 2017 
We resolved this by adding the @JsonFormat annotation which specifies the timezone (EST) for the startDate and endDate column. 
*if timezones were to be changed, modifications would be needed to reflect the new timezone. "

Consistent 1-day offset resurfaced in deployment on v2. Investigation needed.
